sunpy/map/tests/test_mapsequence.py

```diff
@@ -199,6 +200,12 @@ def test_map_sequence_plot(aia171_test_map, hmi_test_map):
     seq.plot()
 
 
+@figure_test
+def test_map_sequence_plot_custom_cmap_norm(aia171_test_map, hmi_test_map):
+    seq = sunpy.map.Map([aia171_test_map, hmi_test_map], sequence=True)
+    seq.plot(cmap='Greys', norm=ImageNormalize(vmin=0, vmax=100))
```

Comment:
Currently this test passes without the fix. The keywords are only ignored when updating the animation so to fully test this the animation should be updated before returning to pytest-mpl. We probably shouldn't be relying on the `_step` method but it works. Calling it once updates it to the first map (AIA), and on the main branch the keywords are ignored.
```suggestion
    animation = seq.plot(cmap='Greys', norm=ImageNormalize(vmin=0, vmax=100))
    animation._step()
    return animation
```

Reply:
Looks like I broke it?

Reply:
My fault for not testing with pytest-mpl enabled. Removing the return will make it pass the current figure (plt.gcf()) to pytest-mpl rather than the animator.

Reply:
That fixed it, I updated the hashes too. Hopefully this will pass now.